indico/modules/events/abstracts/controllers/abstract_list.py

```diff
@@ -107,11 +110,34 @@ def _process(self):
 
 
 class RHCreateAbstract(RHAbstractListBase):
-    def _process(self):
+    """Create an abstract from scratch or with existing cloned data"""
+
+    @staticmethod
```

Comment:
I don't think there's any benefit in marking it as static since we never use it in a static context..